config/groovy/util.groovy

```diff
@@ -214,6 +242,9 @@ def printUsage() {
     println "'-condensed-list-format' to group items by starting letter for the 'list' sub-command (default with many items)"
     println ""
     println "Example: 'groovyw module get Sample -remote jellysnake' - would retrieve Sample from jellysnake's Sample repo on GitHub."
+    println "Example: 'groovyw module get *' - would retrieve all the modules in the Terasology organisation on GitHub."
+    println "Example: 'groovyw module get *de*' - would retrieve all the modules in the Terasology organisation on GitHub" +
```

Comment:
Maybe throw in an example of the `?` syntax here? Possibly instead of / merged into this `*de*` example to save space

Also, if the Windows/Unix case remains a concern that should probably be spelled out here. If a Linux user wants to retrieve all modules would they have to enter `./groovyw module get '*'` ?

Reply:
I've merged the examples into each other and have added a message about the Unix issue.